kartothek/io/dask/delayed.py

```diff
@@ -415,9 +416,11 @@ def read_table_as_delayed(
 
     Parameters
     ----------
+    table: str, optional
+    The table to be loaded. If none is specified, the default 'table' is used.
     """
     if table is None:
-        raise TypeError("Parameter `table` is not optional.")
+        table = SINGLE_TABLE
```

Comment:
Why not just give `SINGLE_TABLE` as default value for the `table` parameter instead of doing this check for `None`?

Reply:
ups... fixed it.